pypeerassets/__main__.py

```diff
@@ -54,6 +54,8 @@ def deck_parser(args: Union[Provider, dict, int, str], prod: bool=True) -> Deck:
     except (InvalidDeckSpawn, InvalidDeckMetainfo, InvalidDeckVersion, InvalidNulldataOutput) as err:
         pass
 
+    raise Exception("undefined behavior :.(")
```

Comment:
This should not be raised, the exceptions above are to allow parser to continue.

Reply:
mmk, i'll update the return type to be Optional[] so it can return None :)